code/modules/overmap/ships/computers/helm.dm

```diff
@@ -0,0 +1,261 @@
+/datum/computer_file/data/waypoint
+	var/list/fields = list()
+
+/obj/machinery/computer/ship/helm
+	name = "helm control console"
+	icon_screen = "helm"
+	light_color = "#7faaff"
+	var/autopilot = 0
+	var/list/known_sectors = list()
+	var/dx		//desitnation
+	var/dy		//coordinates
+	var/speedlimit = 1/(20 SECONDS) //top speed for autopilot, 5
+	var/accellimit = 0.001 //manual limiter for acceleration
+
+/obj/machinery/computer/ship/helm/Initialize()
+	. = ..()
+	get_known_sectors()
+
+/obj/machinery/computer/ship/helm/proc/get_known_sectors()
+	var/area/overmap/map = locate() in world
```

Comment:
AAAAAAAAAAAAAAAAAAA. This will lag the server if we ever EVER spawn this object mid-round.